modules/appnexusBidAdapter.js

```diff
@@ -31,7 +30,7 @@ const SOURCE = 'pbjs';
 export const spec = {
   code: BIDDER_CODE,
   aliases: ['appnexusAst', 'brealtime', 'pagescience', 'defymedia', 'gourmetads', 'matomy', 'featureforward', 'oftmedia', 'districtm'],
-  supportedMediaTypes: [VIDEO, NATIVE],
+  supportedMediaTypes: [BANNER, VIDEO, NATIVE],
```

Comment:
@matthewlane If i am not wrong all other adapters supporting video or native also needs to be updated. As of now https://github.com/prebid/Prebid.js/blob/master/modules/rubiconBidAdapter.js#L76 Rubicon will not work for banner

Reply:
Thanks, looking in to this

Reply:
Good callout @jaiminpanchal27. I went through all adapters that have `supportedMediaTypes` and added 'banner' to those that do support it in addition to 'native' or 'video'. Some adapters only support 'native' or 'video', determined by looking at code and corresponding .md files, and in those cases `supportedMediaTypes` are left unchanged